app/frontend/stylesheets/all/bootstrap_variables.scss

```diff
@@ -98,7 +98,7 @@ $blue: #005dc0 !default;
 // // Quickly modify global styling by enabling or disabling optional features.
 
 // $enable-caret:              true !default;
-$enable-rounded: false;
+// $enable-rounded: false;
```

Comment:
Suspect this affects all bootstrap components across the app. If thats intended its a subtle non-breaking change so should be fine.